src/main/java/frc/robot/commands/Autos.java

```diff
@@ -3,16 +3,10 @@
 // the WPILib BSD license file in the root directory of this project.
 
 package frc.robot.commands;
-
-import frc.robot.subsystems.ExampleSubsystem;
-import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.Commands;
+//import edu.wpi.first.wpilibj2.command.Command;
+//import edu.wpi.first.wpilibj2.command.Commands;
```

Comment:
in general, delete code - don't comment it out. `git` maintains a revision history for you, so commented code just clutters up the files and makes them less readable. there may be other places where this is happening, so double check the other files.